src/nvim/spell.c

```diff
@@ -2889,6 +2890,9 @@ void spell_suggest(int count)
     msg_col = 0;
     // Ask for choice.
     selected = prompt_for_number(&mouse_used);
+
+    ui_call_msg_clear();
```

Comment:
one more thing, we want `if ui_has(kUiMessages)` around it :)

Reply:
Should I also fix the linting issues around my patch or leave it alone?

Reply:
yes, if it is not too much :)